src/components/SearchForm.jsx

```diff
@@ -0,0 +1,31 @@
+import PropTypes from 'prop-types';
+
+const SearchForm = ({ query, setQuery }) => (
+  <div className="border-black-300 bg-rose-400 px-4 py-3">
+    <form
+      data-testid="search-form"
+      className="flex gap-x-0 justify-between items-center"
+    >
+      <input
+        type="text"
+        value={query}
+        onChange={(e) => setQuery(e.target.value)}
+        className="border rounded-md flex-1 h-full p-2 px-3"
+        placeholder="Search by Country Name"
+        data-testid="search-input"
+      />
+      <button
+        type="button"
+        className="bg-blue-600 rounded-md text-white px-3 py-2"
+      >
+        Search
+      </button>
+    </form>
+  </div>
```

Comment:
great work by implementing the search bar as well as by making it a separate component 👍🏼  but please make sure to stick the design guidlines use the same colors for the background as well as for the button 

![image (2)](https://github.com/martinkarugaba/react-redux-capstone/assets/59707859/c978252e-9cf5-4c19-a9fd-31156157834d)